qemu/tests/cfg/timerdevice.cfg

```diff
@@ -90,11 +90,40 @@
                             vcpu_socket = 2
         - newer_msrs_support_check:
             only Linux
-            no Host_RHEL.m5, Host_RHEL.m6.u0
+            no Host_RHEL.m6.u0
             no RHEL.6.0, RHEL.6.1, RHEL.6.2, RHEL.6.3
             type = timerdevice_kvmclock_newer_msrs_support
             rtc_base = utc
             rtc_clock = host
             rtc_drift = slew
             msrs = "4b564d01 4b564d00"
             msrs_catch_re = "kvm-clock: Using msrs (\w+) and (\w+)"
+        - measure_time_jump:
+            only Linux
+            type = timerdevice_time_jump_check
+            rtc_base = utc
+            rtc_clock = host
+            i386, x86_64:
+                rtc_drift = slew
+            check_cmd = "for i in `seq 20`; do date +'%y-%m-%d %H:%M:%S';done"
+            host_cpu_cnt_cmd = "cat /proc/cpuinfo | grep "physical id" | wc -l"
+        - set_time_back:
+            only Linux
+            type = timerdevice_host_time_back
+            ntp_server = clock.redhat.com
+            clock_sync_command = "(service chronyd stop || service ntpdate stop)"
+            clock_sync_command += " && chronyd -q 'server clock.redhat.com iburst'"
+            rtc_base = utc
+            rtc_clock = host
+            i386, x86_64:
+                rtc_drift = slew
+            extra_param = '-global kvm-pit.lost_tick_policy=discard'
+            seconds_to_back = 1800
+            set_host_time_back_cmd = 'date -s "-${seconds_to_back} seconds"'
+            host_epoch_time_cmd = 'epoch=$(date +%s); datetime=$(date);'
+            host_epoch_time_cmd += 'echo "datetime: $datetime epoch: $epoch"'
+            guest_epoch_time_cmd = ${host_epoch_time_cmd}
```

Comment:
I think this is unnecessary since they are totally same. Use one single `epoch_time_cmd` would be enough.